tmt/steps/prepare/distgit.py

```diff
@@ -220,8 +220,8 @@ def go(
 
         try:
             spec_name = next(Path(source_dir).glob('*.spec')).name
-        except StopIteration:
-            raise tmt.utils.PrepareError(f"No '*.spec' file found in '{source_dir}'")
+        except StopIteration as exc:
```

Comment:
`StopIteration` as kind of special errors, they just mean "nothing else in this iterator", and as such they don't bring much of an added value as causes. It's nicely consistent, but it's mostly likely not providing anything above "yep, we run out of items". Your call.